src/app/players/collection-player/collection-details-model/collection-details-model.component.spec.ts

```diff
@@ -30,4 +30,9 @@ describe('CollectionDetailsModelComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+  it('#ngOnInit() should define viewPropertiesList data ', () => {
+    component.ngOnInit();
+    expect(component.viewPropertiesList).toBeDefined();
+    expect(component.viewPropertiesList[0].value).toBeDefined();
```

Comment:
@santoshilimi No need to check inside value of the variable. so you can remove this line as already covered by above line.